src/chrome/content/rules/sncf.com.xml

```diff
@@ -0,0 +1,39 @@
+<ruleset name="SNCF (Partial)">
+  <target host="authentification.sncf.com"/>
+  <target host="bourselogement.sncf.com"/>
+  <target host="challenge.sncf.com"/>
+  <target host="compte.sncf.com"/>
+  <target host="data.sncf.com"/>
+  <target host="echange.sncf.com"/>
+  <target host="extranet.sncf.com"/>
+  <target host="fidelite-programme-voyageur.sncf.com"/>
+  <target host="goodies.sncf.com"/>
+  <target host="lestropheesexcellence.sncf.com"/>
```

Comment:
This target provides no certificate chain. Please comment it out or split it into a separate default_off ruleset.